### DISCUSSION TITLE
LDAP configuration bind user

### DISCUSSION THREAD

**Discussion by ibrahimdjadir** [dfir-iris/iris-web#266] [Q&A] [answered]
Hello,
I would like to inquire about configuring an LDAP server. I couldn't find the appropriate environment variable to set the bind user for performing searches. Are the variables LDAP_USER_PREFIX and LDAP_USER_SUFFIX used for the bind user? If so, where should I include the password? Additionally, I would like to know if it is possible to configure STARTTLS encryption and, if yes, how to do it. 
Thank you

**Comment by juadde** [comment, marked as answer]
AFAIK, LDAP user itself is used for the bind & search, and LDAP_USER_PREFIX & LDAP_USER_SUFFIX are used to recreate the dn of the user given his username (entered by the user).

As python-ldap3 is used to communicate with the LDAP STARTTLS should be possible but I don't know if the capability has been implemented in IRIS.

**Reply by ibrahimdjadir** [reply]
thank you for your help, I used user self-bind, and it's working. concerning starttls I did not find any env var that activates this option so I use ldaps instead
